### ISSUE TITLE
debile-remote: add commande update-builder

### ISSUE THREAD

**Issue by clemux** [opencollab/debile#39]
debile-remote should be able to modifiy a builder's configuration

The more urgent functionality is the ability to change a builder ip address (for ip-based authentication).

Relevant files are:
- https://github.com/opencollab/debile/blob/master/debile/utils/cli.py#L51
- https://github.com/opencollab/debile/blob/master/debile/master/interface.py#L181

This diff can give some pointers: https://github.com/opencollab/debile/commit/0fdea93dfe3869c374baee4c05833966daf0144d

Basically:
1. add a "update_builder" handler in the  server interface (debile/master/interface.py)
2. add a "update-builder" command to debile-remote (debile/utils/cli.py)
